scripts/install/install-pocs.sh

```diff
@@ -152,6 +152,8 @@ export PANDIR=${PANDIR}
 export POCS=${PANDIR}/POCS
 export PANLOG=${PANDIR}/logs
 EOF
+
+    echo '. /var/panoptes/env' >> ~/.bashrc
```

Comment:
A slightly better trick (that I just learned) that will check for the existence of the file first is:

```suggestion
    [ ! -f /var/panoptes/env ] && echo '. /var/panoptes/env' >> ~/.bashrc
```